src/datagrid/DataGrid.tsx

```diff
@@ -744,23 +760,47 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     // function to render select cell
     private buildSelectCell(rowID: number, isSelected: any): React.ReactElement {
         const {selectionType} = this.props;
-        if (selectionType === GridSelectionType.MULTI) {
-            return (
-                <CheckBox
-                    id={rowID.toString()}
-                    ariaLabel="Select"
-                    onChange={evt => this.handleSelectSingle(evt, rowID)}
-                    checked={isSelected !== undefined ? isSelected : undefined}
-                />
-            );
-        }
         return (
-            <RadioButton
-                value={rowID}
-                id={rowID.toString()}
-                onChange={evt => this.handleSelectSingle(evt, rowID)}
-                checked={isSelected !== undefined ? isSelected : undefined}
-            />
+            <div className={ClassNames.DATAGRID_ROW_STICKY}>
+                <div
+                    role="gridcell"
```

Comment:
Can we move these hardcoded values to constants or enum.

Reply:
@maheshkulkarni4 those are part of DOM there is no use of this in code logic so not converting it to enum. Clarity may have some CSS associated with it.